config/capability_configuration.go

```diff
@@ -49,24 +49,18 @@ func (c CapabilityConfigurations) Validate(ic core.IacContext, pc core.ObjectPat
 	return nil
 }
 
-func (c CapabilityConfigurations) ToCapabilities(stackId int64) []types.Capability {
```

Comment:
Is this removed because we no longer store the stackId in the config? Because it is now in the effective config?configuration?
This certainly simplifies this but make sure the functionality isn't lost.

Reply:
I improved furion logic to use the owning block's stack id if a capability did not have a stack id.
This is effectively dead code.

Reply:
See this change: https://github.com/nullstone-io/furion/pull/129/files#diff-b78717981029b1a31ae3d9a8b7ce60b475015667a3aec4c43bc8d3ef0911b685R12-R15